lib/rules/require-computed-property-dependencies.js

```diff
@@ -224,8 +224,11 @@ function findThisGetCalls(node) {
   new Traverser().traverse(node, {
     enter(child, parent) {
       if (
-        types.isMemberExpression(child) &&
-        !(types.isCallExpression(parent) && parent.callee === child) &&
+        (types.isOptionalMemberExpression(child) || types.isMemberExpression(child)) &&
+        !(
+          (types.isCallExpression(parent) || parent.type === 'OptionalCallExpression') &&
```

Comment:
Should there be a `types.isOptionalCallExpression` helper?

Reply:
Fixed.